server/models/BookingRequest.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const bookingRequestSchema = new mongoose.Schema({
+    user_id: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true,
+        ref: "User"
+    },
+    sitter_id: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true,
+        ref: "User"
+    },
+    start: {
+        type: Date,
+        default: Date.now,
+        required: true
+    },
+    end: {
+        type: Date,
+        default: Date.now,
+        required: true
+    },
+    description: {
+        type: String,
+        default: '',
+    },
+    accepted: {
```

Comment:
Optional: You can use an 'enum' object to combine the accepted/declined fields

Eg: 
```
status: {
type: String,
enum: ['pending', 'accepted', 'declined']
default: 'pending'
required: true,
}
```

Reply:
resolved by 100d2625ece5229da983eb7c6d61efd8b6ef3809